README.md

```diff
@@ -33,4 +45,3 @@ Lombok has a lot of useful annotations that give you some of the nice things in
 ### Efficient Primitive Collections
```

Comment:
They generally out-perform for large hash maps as well as primitive collections compared to JDK, Scala, Guava, and Trove. The repo has performance tests comparing against Guava, Trove, etc. (see [here](https://github.com/goldmansachs/gs-collections/tree/master/performance-tests)). The primary reason I use the code is that it just has more collections than anyone as well as a range of types of iterators (external/internal/eager/lazy/etc.).  It's very Java 8 friendly in terms of using functional interfaces.